pipeline-model-definition/src/main/groovy/org/jenkinsci/plugins/pipeline/modeldefinition/validator/ModelValidatorImpl.groovy

```diff
@@ -455,6 +456,14 @@ class ModelValidatorImpl implements ModelValidator {
                         }
                         ModelASTKeyValueOrMethodCallPair kvm = (ModelASTKeyValueOrMethodCallPair) a
 
+                        // Special case, see JENKINS-63499.
+                        if (model.type == PasswordParameterDefinition.class && "defaultValue".equals(kvm.key.key)) {
+                            if (!validateParameterType((ModelASTValue) kvm.value, String.class, kvm.key)) {
```

Comment:
The validator is smart and checks to make sure that all parameters exist and have the expected types. This is a workaround because `model.getParameter('defaultValue')` will return `null` on Jenkins 2.236 and newer causing validation to fail. On earlier versions of Jenkins, this is just a shortcut for the check that would have happened anyway.